.github/workflows/go.yml

```diff
@@ -50,7 +56,14 @@ jobs:
       if: runner.os == 'Windows'
       run: choco install -y make mingw
 
+    - name: Setup
+      run: make setup
```

Comment:
Docker isn't supported on mac/windows Github runners. We should guard this step. 

https://github.com/moov-io/achgateway/blob/master/.github/workflows/go.yml#L38